tests/integration/api/controllers/person.spec.js

```diff
@@ -133,33 +130,44 @@ describe('Person API', () => {
         ...onlineUserPlaceHierarchy
       }
     };
+    const expectedPeople = [
+      contact0,
+      contact1,
+      contact2,
+      patient,
+      e2eTestUser,
+      {
+        type: personType,
+        ...userNoPerms.contact,
+        ...onlineUserPlaceHierarchy
+      },
+      {
+        type: personType,
+        ...offlineUser.contact,
+        ...offlineUserPlaceHierarchy
+      }
+    ];
 
-    it('returns a page of people', async () => {
-      const expectedPeople = [
-        contact0,
-        contact1,
-        contact2,
-        patient,
-        e2eTestUser,
-        {
-          type: personType,
-          ...userNoPerms.contact,
-          ...onlineUserPlaceHierarchy
-        },
-        {
-          type: personType,
-          ...offlineUser.contact,
-          ...offlineUserPlaceHierarchy
-        }
-      ];
-      const responsePage = await getPage(Qualifier.byContactType(personType), cursor, limit);
+    it('returns a page of people for no limit and cursor passed', async () => {
+      const responsePage = await getPage(Qualifier.byContactType(personType));
       const responsePeople = responsePage.data;
       const responseCursor = responsePage.cursor;
 
       expect(responsePeople).excludingEvery(['_rev', 'reported_date']).to.deep.equalInAnyOrder(expectedPeople);
       expect(responseCursor).to.be.equal(null);
     });
 
+    it('returns a page of people when limit and cursor is passed and cursor can be reused', async () => {
+      const firstPage = await getPage(Qualifier.byContactType(personType), cursor, limit);
+      const secondPage = await getPage(Qualifier.byContactType(personType), firstPage.cursor, limit);
+
+      const allPeople = [...firstPage.data, ...secondPage.data];
+
+      expect(allPeople).excludingEvery(['_rev', 'reported_date']).to.deep.equalInAnyOrder(expectedPeople);
+      expect(firstPage.cursor).to.be.equal('4');
+      expect(secondPage.cursor).to.be.equal(null);
```

Comment:
Minor, but can you also assert the `data.length` for each page?  Just want to confirm that we got `4` back on the first page and `3` on the second.